scriptmodules/ports/hexen2.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
+
+# This file is part of The RetroPie Project
+#
+# The RetroPie Project is the legal property of its developers, whose names are
+# too numerous to list here. Please refer to the COPYRIGHT.md file distributed with this source.
+#
+# See the LICENSE.md file at the top-level directory of this distribution and
+# at https://raw.githubusercontent.com/RetroPie/RetroPie-Setup/master/LICENSE.md
+#
+
+rp_module_id="hexen2"
+rp_module_desc="Hexen II - Hammer of Thyrion source port"
+rp_module_licence="GLP2"
+rp_module_help="For registered version, please add your full version PAK files to $romdir/ports/hexen2/data1/ to play. These files for the registered version are required: pak0.pak, pak1.pak and strings.txt. The registered pak files must be patched to 1.11 for Hammer of Thyrion."
+rp_module_section="opt"
+rp_module_flags=""
+
+
+# Get depends required by hexen2
```

Comment:
please removed these comments when code complete.

Reply:
Didnt know that the comments needs to be removed. I will do that in a newer revised version. Thanks.

Reply:
It has been solved in new commits (revised version).